ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.component.ts

```diff
@@ -246,21 +246,25 @@ export class NetworksComponent implements OnInit {
         const httpsProxyValue = this.form.get('httpsProxy').value;
         const httpsProxyPortValue = this.form.get('httpsProxyPort').value;
 
-        results['containerNetworks'] = this.form.get('containerNetworks')
-            .value
-            .filter(entry => entry['containerNetwork']);
+        if (this.inAdvancedMode) {
+            results['containerNetworks'] = this.form.get('containerNetworks')
```

Comment:
At some point we should be getting/setting this from a viewModel rather than directly from the form. But we can do this later during refactor.